builders/build/canonical-contracts/precompiles/batch.md

```diff
@@ -0,0 +1,225 @@
+
+Instead of deploying the Batch precompile, you will access the interface given the address of the precompiled contract:
+
+1. Click on the **Deploy and Run** tab directly below the **Compile** tab in Remix. Please note the precompiled contract is already deployed
+2. Make sure **Injected Web3** is selected in the **Environment** dropdown. Once you select **Injected Web3**, you might be prompted by MetaMask to connect your account to Remix
+3. Make sure the correct account is displayed under **Account**
+4. Ensure **Batch - Batch.sol** is selected in the **Contract** dropdown. Since this is a precompiled contract, there is no need to deploy any code. Instead we are going to provide the address of the precompile in the **At Address** Field
+5. Provide the address of the Batch precompile: `{{networks.moonbase.precompiles.batch}}` and click **At Address**
+
+![Access the address](/images/builders/build/canonical-contracts/precompiles/batch/batch-2.png)
+
+The **IERC20** precompile will appear in the list of **Deployed Contracts**.
+
+### Deploy SimpleMessage
+
+On the other hand, SimpleMessage.sol will be deployed as a new contract. Before starting this section, repeat the [compilation step](#compile-the-contract) with the **SimpleMessage.sol** file.
+
+1. Click on the **Deploy and Run** tab directly below the **Compile** tab in Remix
+2. Make sure **Injected Web3** is selected in the **Environment** dropdown. Once you select **Injected Web3**, you might be prompted by MetaMask to connect your account to Remix
+3. Make sure the correct account is displayed under **Account**
+4. Ensure **SimpleMessage - SimpleMessage.sol** is selected in the **Contract** dropdown
+5. Click **Deploy**
+6. Confirm the Metamask transaction that appears by clicking **Confirm**
+
+![Deploy SimpleMessage](/images/builders/build/canonical-contracts/precompiles/batch/batch-3.png)
+
+The **SimpleMessage** contract will appear in the list of **Deployed Contracts**.
+
+### Send Native Currency via Precompile {: #send-native-currency-via-precompile }
+
+Sending native currency with the batch precompile is more involved than pressing a few buttons in Remix or Metamask. 
+
+Transactions have a value field to specify the amount of native currency being sent with it. In Remix, this is represented by the **VALUE** input in the **DEPLOY & RUN TRANSACTIONS** tab. However, for the batch precompile, this data is provided within the *value* array input of the batch functions.
+
+Try transferring native currency to two wallets of your choice via the batch precompile on Moonbase Alpha:
+
+1. Make sure that you have at least 0.5 DEV in your connected wallet
+2. Expand the Batch contract under **Deployed Contracts**
+3. Expand the **batchAll** function
+4. For the *to* input, insert your addresses in the following format: `["0xADDRESS1", "0xADDRESS2"]`, where 0xADDRESS1 is the address of the first wallet of your choice and 0xADDRESS2 is the address of the second wallet of your choice
```

Comment:
Yes, addresses typically have the 0x, I just included it there because I figured that it would emphasize the fact that it was an address. 
I think generally the other tutorials just use a predefined address, but I didn't want people to lose their DEV for the sake of the tutorial. But if @eshaben thinks its better to use a different format or tutorial content, then I could easily change it